src/main/java/io/micronaut/build/MicronautDevelocityPlugin.java

```diff
@@ -0,0 +1,158 @@
+            boolean remoteEnabled = micronautBuildSettingsExtension.getUseRemoteCache().get();
+            boolean push = MicronautBuildSettingsExtension.booleanProvider(providers, "cachePush", isCI).get();
+            if (isCI) {
+                System.out.println("Build cache     enabled     push");
+                System.out.println("    Local        " + (localEnabled ? "   Y   " : "   N   ") + "     N/A");
+                System.out.println("    Remote       " + (remoteEnabled ? "   Y   " : "   N   ") + "     " + (push ? " Y" : " N"));
+            }
+            buildCache.getLocal().setEnabled(localEnabled);
+            if (remoteEnabled) {
+                configureRemoteCache(config, buildCache, push, providers);
+            }
+        });
+    }
+
+    private static void configureRemoteCache(DevelocityConfiguration config, BuildCacheConfiguration buildCache, boolean push, ProviderFactory providers) {
+        buildCache.remote(config.getBuildCache(), remote -> {
+            remote.setEnabled(true);
+            if (push) {
+                String accessKey = providers.systemProperty("GRADLE_ENTERPRISE_ACCESS_KEY").getOrNull();
+                if (accessKey != null && !accessKey.isEmpty()) {
+                    remote.setPush(true);
+                } else {
+                    System.err.println("WARNING: Access key missing for remote build cache, cannot configure push!");
+                }
+            }
+        });
+    }
+
+    private void tagWithJavaDetails(DevelocityConfiguration ge, ProviderFactory providers) {
+        var vendor = providers.systemProperty("java.vendor");
+        if (vendor.isPresent()) {
+            ge.getBuildScan().tag("vendor:" + vendor.get().toLowerCase().replaceAll("\\W+", "_"));
+        }
+        ge.getBuildScan().tag("jdk:" + JavaVersion.current().getMajorVersion());
+    }
+
+    private void captureSafeEnvironmentVariables(DevelocityConfiguration ge) {
+        for (var variable : SAFE_TO_LOG_ENV_VARIABLES) {
+            String value = System.getenv(variable);
+            ge.getBuildScan().value("env." + variable, value == null ? "<undefined>" : value);
+        }
+    }
+
+    private void configureBuildScansPublishing(DevelocityConfiguration config,
+                                               boolean isCI,
+                                               Provider<Boolean> publishScanOnDemand,
+                                               boolean buildScanExplicit) {
+        config.getServer().convention("https://ge.micronaut.io");
+        config.buildScan(buildScan -> {
+            buildScan.getPublishing().onlyIf(context -> {
+                if (Boolean.TRUE.equals(publishScanOnDemand.get())) {
+                    return buildScanExplicit;
+                }
+                return  isCI || context.isAuthenticated();
+            });
+            buildScan.capture(c -> c.getFileFingerprints().convention(true));
```

Comment:
💬 You don't need this anymore, as `fileFingerprints` is `true` by default: https://docs.gradle.com/develocity/gradle-plugin/legacy/#updated_file_fingerprints_capturing_api

Reply:
Thanks! Removed.